monorepo-migration/sync-fenix.sh

```diff
@@ -55,13 +53,13 @@ function _setup_temporary_repo() {
 
     git clone "git@github.com:mozilla-mobile/$REPO_NAME_TO_SYNC.git" "$TMP_REPO_PATH"
     cd "$TMP_REPO_PATH"
-    git fetch origin "$TMP_REPO_BRANCH_NAME"
+    #git fetch origin "$TMP_REPO_BRANCH_NAME"
 }
 
 function _update_repo_branch() {
     git checkout "$TMP_REPO_BRANCH_NAME"
     git rebase main
-    git push origin "$TMP_REPO_BRANCH_NAME" --force
+    #git push origin "$TMP_REPO_BRANCH_NAME" --force
```

Comment:
Nit: No need to comment out this line anymore since we're already skipping this function in: 

https://github.com/mozilla-mobile/firefox-ios/pull/18727/files#diff-26f75150de3654539f07404abeb474003f616baa09620e854e587bfc611d1a85R128